packages/dashboard/src/hooks/useImageFailOver.ts

```diff
@@ -0,0 +1,14 @@
+import { useAsync } from 'react-use'
+import { findAvailableImageURL } from '@dimensiondev/kit'
+
+export function useImageFailOver(urls: readonly string[], suffix: string) {
```

Comment:
Please give a comment. Quite hard to figure out what is it used for

Reply:
`useImageFailover` is the hook in `packages/maskbook`, here it is just a replica

Reply:
I mean comment in the code